src/ucp/rndv/rndv.h

```diff
@@ -19,7 +19,10 @@ typedef enum {
     UCP_RNDV_RTS_FLAG_TAG_COMPAT_VALUE_OK       = UCS_OK,
     UCP_RNDV_RTS_FLAG_TAG_COMPAT_VALUE_CANCELED = (uint8_t)UCS_ERR_CANCELED,
     UCP_RNDV_RTS_FLAG_TAG                       = UCS_BIT(0),
-    UCP_RNDV_RTS_FLAG_AM                        = UCS_BIT(1)
+    UCP_RNDV_RTS_FLAG_AM                        = UCS_BIT(1),
+    /* Other bits mustn't be used for RNDV RTS flags to avoid conflicting with
```

Comment:
"Other" looks confusing, pls replace with "UCP_RNDV_RTS_FLAG_MAX and higher bits ..."

Reply:
fixed